packages/ag-charts-enterprise/src/features/crosshair/crosshairLabel.css

```
@@ -13,14 +13,14 @@
     box-sizing: border-box;
 }
 
-.ag-crosshair-label-content {
+.ag-charts-crosshair-label-content {
     padding: 0 8px;
     border-radius: 2px;
     line-height: 20px;
     background-color: var(--ag-charts-axis-label-background-color);
     color: var(--ag-charts-axis-label-color);
 }
 
-.ag-crosshair-label-hidden {
+.ag-charts-crosshair-label-hidden {
```

If it's not too much hassle, I think we should still keep the modifiers clear (this is what I did in tooltips)

```suggestion
.ag-charts-crosshair-label--hidden {
```

---

Applied